tests/testthat/create-table-bug.R

```diff
@@ -0,0 +1,8 @@
+## pre-bugfix : this blows up with the "Create Table" bug. 
+
+go <- function () {
+    fn <- function (n) data.frame(x = 1:n, y = runif(n))
+    f2 <- cache(fn, key = 'n', salt = c(), con = "~/dev/cachemeifyoucan/tests/test-resources/database.yml", env = "test.local", prefix = "F2")
```

Comment:
Agreed, use the [db helper](https://github.com/robertzk/cachemeifyoucan/blob/master/tests/testthat/helper-db.R)